app/src/Commands/Repo/RepoManager.php

```diff
@@ -156,13 +173,30 @@ public function getLocalRepo(string $name) : Repo
         });
 
         if (! count($repos)) {
-            throw new RuntimeException("Repo $name not found in local repos.");
+            throw new RuntimeException("Repo with name '$name' not found in local repos.");
```

Comment:
This is slightly different to the message in getGlobalRepoByName(), might be better if they are both phrased the same way (also needs to be changed in the test result).

Reply:
done!